CHANGELOG.md

```diff
@@ -66,6 +66,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## 4.2.0 (2018-07-18)
```

Comment:
Should this change be under a new `4.4.3 (Unreleased)` section instead?

Reply:
done